docs/preview/continuous-patching/README.md

```diff
@@ -0,0 +1,477 @@
+        "repository": "<Repository Name>",
+        "tags": ["<comma-separated-tags>"],   
+        "enabled": <true|false>
+    }] 
+}
+```
+The schema ingests specific repositories and tags in an array format. Each variable is defined below:
+
+- "version" allows the ACR team to track what schema version you’re on. Do not change this variable unless instructed to.
+- "tag-convention" this is an optional field. Allowed values are "incremental" or "floating" - refer to **Key Concepts** for more information.
+
+- "repositories" is an array that consists of all objects that detail repository and tag information
+    - "repository" refers to repository name
+    - "tags" is an array of tags separated by commas. The wildcard "*" can be used to signify all tags within that repository
+    - "enabled" is a Boolean value of true or false determining if the specified repo is on or off
+
+The following details an example configuration for a customer who wants to patch all tags (use the * symbol) within the repository “python”, and to patch specifically the “jammy-20240111” and “jammy-20240125” tags in the repository “ubuntu”. 
+
+JSON example:
+```json
+{
+"version": "v1",
+"tag-convention" : "incremental",
+"repositories": [{
+        "repository": "python",
+        "tags": ["*"],
+        "enabled": true
+    },
+    {
+        "repository": "ubuntu",
+        "tags": ["jammy-20240111", "jammy-20240125"],
+        "enabled": true, 
+    }]
+}
+```
+4. After creating your configuration file, it is recommended to execute a dry run to verify the intended artifacts are selected by the JSON criteria. The dry run requires a parameter called schedule, which specifies how often your continuous patching cycle will run. The schedule flag is measured in days, with a minimum value of 1 day, and a maximum value of 30 days. For example, if you want an image to be patched everyday, you would specify schedule as "1d", or 1 day. If you want a weekly patch (once a week), you would fill schedule as "7d", or 7 days. 
+
+Command Schema:
+```sh
+az acr supply-chain workflow create -r <registryname> -g <resourcegroupname> -t continuouspatchv1 --config <JSONfilepath> --schedule <number of days> --dry-run 
+```
+Example Command: 
+```sh
+az acr supply-chain workflow create -r myRegistry -g myResourceGroup -t continuouspatchv1 -–config ./continuouspatching.json --schedule 1d –-dry-run   
+```
+
+The --dry-run flag will output all specified artifacts by the JSON file configuration. Customers can verify that the right artifacts are selected. With the sample ubuntu configuration above, the following results should be displayed as output. 
+```sh
+Ubuntu: jammy-20240111
+Ubuntu: jammy-20240125
+```
+Help command to see all required/optional flags.
+```sh
+az acr supply-chain workflow create --help
+```
+ 
+5. Once satisfied with the dry-run results, run the ‘create’ command again without the --dry-run flag to officially create your continuous patching workflow.  
```

Comment:
Avoid smart quotes for the create command and have `--dry-run` be in inline code.